db/migrate/20230223000336_create_reviews.rb

```diff
@@ -0,0 +1,12 @@
+class CreateReviews < ActiveRecord::Migration[7.0]
+  def change
+    create_table :reviews do |t|
+      t.string :title
+      t.text :comment
+      t.float :rating
+      t.references :art, null: false, foreign_key: true
+
+      t.timestamps
+    end
```

Comment:
No that I think about it, what's a title for a review supposed to be ? 🤔 
I don't really remember seing something like that on a review before
![image](https://user-images.githubusercontent.com/75388869/221056584-5d6bff1d-c2cc-4c8c-8ce4-b888273ae827.png)

Reply:
I would say it could help in giving a very global overview of the review, like a news article title. 
But to keep it light we should maybe take it out.
<img width="829" alt="Screenshot 2023-02-24 at 10 13 55" src="https://user-images.githubusercontent.com/112162648/221139548-64a1e8bb-4165-469a-a684-f05bae474da3.png">

Reply:
That's tripadvisor right ? It's not too bad 🤔